### PR TITLE
Main nav

### PR DESCRIPTION
**Pull request by milenacard**

### CODE REVIEW

File: data/mainNav.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        "label": "Home",
+        "href": "/"
+      },
+      {
+        "label": "About us",
+        "href": "/"
+      }
+    ]
+  }
+}
```
Comment (andrscrrn) [inline data/mainNav.json]: You shouldn't have a file just for an specific component. The idea is to have a file per page.

File: src/components/Main-nav/MainNav.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    this.elements.list = this.node.querySelector('.main-menu__list')
+    this.elements.listItems = this.elements.list.querySelectorAll('.main-menu__list-item')
+    this.elements.itemButtons = this.elements.list.querySelectorAll('.main-menu__list-item-button')
+    this.elements.listLinks = this.elements.list.querySelectorAll('.main-menu__list__link')
+  }
+
+  setEventHamburger () {
+    this.elements.hamburger = this.elements.navBar.querySelector('.main-menu__hamburger')
+    this.elements.hamburger.addEventListener('click', () => {
+      this.elements.hamburger.classList.toggle('main-menu__hamburger--clicked')
```
Comment (andrscrrn) [inline src/components/Main-nav/MainNav.js]: `main-menu__hamburger--clicked` should be a constant.

File: src/components/Main-nav/MainNav.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  showItem (index) {
+    this.elements.listLinks[index].classList.toggle('main-menu__list__link--activate')
+  }
+
+  setEventlist () {
+    this.elements.itemButtons.forEach(element => {
+      element.addEventListener('click', () => {
+        const index = Array.from(this.elements.itemButtons).indexOf(element)
+        if (this.currentListButton === null) {
+          console.log('esta es la primera vez ')
```
Comment (andrscrrn) [inline src/components/Main-nav/MainNav.js]: No console logs.

File: src/components/Main-nav/MainNav.scss [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+		height: 5vh;
+		display: flex;
+		cursor: pointer;
+		font-family: $font-family;
+
+		@include large{
+			width: 100%;
+			height: 100px;
+			position: relative;
+			//background: #f9c833;
```
Comment (andrscrrn) [inline src/components/Main-nav/MainNav.scss]: No commented code.

File: src/components/Main-nav/MainNav.scss [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+			background: #c21500;  /* fallback for old browsers */
+			background: -webkit-linear-gradient(left, #ffc500, #c21500);  /* Chrome 10-25, Safari 5.1-6 */
```
Comment (andrscrrn) [inline src/components/Main-nav/MainNav.scss]: We don't need to use fallbacks in the code level. If you need them, use them in a build level (Webpack).

File: src/components/Main-nav/MainNav.scss [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+			}
+
+			&-last{
+				top: 36px;
+			}
+		}
+
+		&--clicked{
+			.main-menu__hamburger-line{
+				background: #f9c833;
```
Comment (andrscrrn) [inline src/components/Main-nav/MainNav.scss]: Colors should be variables.

**Review by andrscrrn — CHANGES_REQUESTED** [review]

File: src/components/Main-nav/MainNav.js (hunk 2 above)
Comment (milenacard) [reply src/components/Main-nav/MainNav.js]: I don´t understand. Do you say that I change `main-menu__hamburger--clicked` for `main-menu__hamburger-clicked` ?

File: data/data.json [hunk 7]
```diff
@@ -1 +1,69 @@
-{"name": "Milena"}
+{
+  "mainNav": {
+    "logo": {
+      "img": "McDonals",
```
Comment (andrscrrn) [inline data/data.json]: This is not a valid url. How are you currently using this value?

File: src/components/Main-nav/MainNav.scss [hunk 8]
```diff
@@ -0,0 +1,260 @@
...
+		height: 5vh;
+		display: flex;
+		cursor: pointer;
+		font-family: $font-family;
+
+		@include large{
+			width: 100%;
+			height: 100px;
+			position: relative;
+			background: -webkit-linear-gradient(left, $primaryColor, $secondaryColor);
```
Comment (andrscrrn) [inline src/components/Main-nav/MainNav.scss]: You don't need to use prefixed css as the `-webkit-linear-gradien`. If it is really necessary, add a new plugin in the Webpack configuration to help you with that. If you need help with this, please let me know.

File: src/components/shared/_base.scss [hunk 9]
```diff
@@ -0,0 +1,11 @@
+@mixin medium {
+  @media (min-width: 640px) and (max-width: 1023px) {
```
Comment (andrscrrn) [inline src/components/shared/_base.scss]: I would suggest to save in variables the breakpoint values.

**Review by andrscrrn — CHANGES_REQUESTED** [review]

File: data/data.json (hunk 7 above)
Comment (milenacard) [reply data/data.json]: I am using this value like this: `src=mainNav.logo.link`

Comment (andrscrrn) [reply data/data.json]: This comment is about like 4, not 5.

**Review by andrscrrn — CHANGES_REQUESTED** [review]

**Pull request merged**